tempodb/backend/cache/cache.go

```diff
@@ -110,8 +110,8 @@ func (r *readerWriter) CloseAppend(ctx context.Context, tracker backend.AppendTr
 	return r.nextWriter.CloseAppend(ctx, tracker)
 }
 
-func (r *readerWriter) Delete(context.Context, string, backend.KeyPath) error {
-	panic("delete is not supported for cache.Cache backend")
+func (r *readerWriter) Delete(ctx context.Context, name string, keypath backend.KeyPath) error {
+	return r.nextWriter.Delete(ctx, name, keypath)
```

Comment:
we can't really do this. the point of the panic was to highlight that the cache does not support deleting. this is making it silently fail which means we will create some difficult to detect bugs in the future.

Reply:
This has to do with the reads for these objects will still be cached?